Composer/packages/client/src/recoilModel/atoms/appState.ts

```diff
@@ -235,6 +235,11 @@ export const formDialogGenerationProgressingState = atom({
   default: false,
 });
 
+export const formDialogErrorState = atom<(Error & { kind: 'templateFetch' | 'generation' | 'deletion' }) | undefined>({
```

Comment:
Nit: I thought you discouraged these anonymous param types ;)

Reply:
the union type doesn't need import (++ for union type vs. enum) so this inline type never needs to be imported anywhere.